addons/mrp_subcontracting/views/subcontracting_portal_templates.xml

```diff
@@ -19,10 +19,13 @@
     <template id="portal_my_home_menu_production" name="Portal layout : production menu entries" inherit_id="portal.portal_breadcrumbs" priority="25">
         <xpath expr="//ol[hasclass('o_portal_submenu')]" position="inside">
             <t t-if="title">
-                <li t-if="page_name == 'production' or production" t-attf-class="breadcrumb-item #{'active' if breadcrumbs_url else ''}">
+                <li t-if="page_name == 'production' or picking" t-attf-class="breadcrumb-item #{'active' if breadcrumbs_url else ''}">
```

Comment:
why did you change this ?

Reply:
The breadcrumb is currently broken in Master, doing that change seems to fix its behavior. Is there something I'm not aware of ?